WebViewControl.Avalonia/WebViewControl.Avalonia.csproj

```diff
@@ -13,7 +13,7 @@
     <Copyright>Copyright ©  2019</Copyright>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Version>2.91.21</Version>
+    <Version>2.91.23</Version>
```

Comment:
Had to bump to 2.91.23 because I released 2.91.22 package with prior fix approach.